uktrade/data-hub-frontend
# Timeline component...

## Description of change

We need a component that gives the user an indication of the progress of an investment project. We have a version of this in Nunjucks but not React and this will be used for the new dashboard.

Note: Don't take too much notice of the styling as this is subject to change.

## Test instructions
- View Storybook and you should see the new timeline component
- should respond from mobile to desktop

## Screenshots
![Screenshot 2021-02-03 at 14 37 00](https://user-images.githubusercontent.com/10154302/106762305-9c7e0c80-662d-11eb-9feb-4f4f39e7fa5a.png)
![Screenshot 2021-02-03 at 14 37 15](https://user-images.githubusercontent.com/10154302/106762323-a142c080-662d-11eb-9f8e-54540e32b006.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
